test/functional/core/job_spec.lua

```diff
@@ -14,6 +14,7 @@ local nvim_set = helpers.nvim_set
 local expect_twostreams = helpers.expect_twostreams
 local expect_msg_seq = helpers.expect_msg_seq
 local Screen = require('test.functional.ui.screen')
+local nvim_cat = iswin() and nvim_dir..'\\cat' or 'cat'
```

Comment:
This should not be needed. Other tests are already using cat.exe . Though maybe they were using cmd.exe instead of powershell.

Reply:
I don't know why the flaky tests are passing. Which cat comes firsts in `PATH`?